tests/conftest.py

```diff
@@ -863,7 +863,7 @@ def towns_fund_round_6_file_success() -> Generator[BinaryIO, None, None]:
 def towns_fund_round_6_file_failure() -> Generator[BinaryIO, None, None]:
     """An example spreadsheet for reporting round 6 of Towns Fund that should ingest with validation errors."""
     with open(
-        Path(__file__).parent / "integration_tests" / "mock_tf_returns" / "TF_Round_6_Project_Progress_Failure.xlsx",
+        Path(__file__).parent / "integration_tests" / "mock_tf_returns" / "TF_Round_6_General_Validation_Failures.xlsx",
```

Comment:
The `TF_Round_6_General_Validation_Failures.xlsx` has similar issues with the other `TF_Round_6` test files: it's not up to date with the latest changes. It does not break things, but neither does it test them properly.

I hoping to get a R6 success sheet soon from Craig. We should get the R6 initial/general validation failure sheets fixed too.